examples/src/main/scala/sttp/tapir/examples/MultipleEndpointsDocumentationHttp4sServer.scala

```diff
@@ -43,6 +42,7 @@ object MultipleEndpointsDocumentationHttp4sServer extends App {
   implicit val ec: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
   implicit val contextShift: ContextShift[IO] = IO.contextShift(ec)
   implicit val timer: Timer[IO] = IO.timer(ec)
+  implicit val concurrent: Concurrent[IO] = IO.ioConcurrentEffect
```

Comment:
why do we need a concurrent instance now, as we didn't need one before?

Reply:
It is used implicitly by Http4sServerToHttpInterpreter's apply method

Reply:
But ... it wasn't needed before, so it shouldn't be needed now?

Reply:
You are right I am missing type parameter for Http4sServerInterpreter